.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release Krakend helm chart
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout tag
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Publish Helm Chart
+        uses: tylerauerbeck/helm-gh-pages@main
+        with:
+          token: ${{ secrets.PAT }}
```

Comment:
do we need to use a PAT here? Can't we use the `$GITHUB_TOKEN` that's already available to the workflow?

Reply:
Oh, do you know which user that token is associated with? Also is it just `secrets.GITHUB_TOKEN`?

Reply:
@pereztr5 it's a temporary user that has the current workflow as the subject. You're able to add/remove permissions to it depending on what you wanna do [1]. I guess it really depends on where the target repo is.

[1] https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings

Reply:
@sfunkhouser got this token, `PUBLIC_REPO_GH_PAT`,  working for this repo.